plugins/de.cognicrypt.codegenerator/src/main/java/de/cognicrypt/codegenerator/wizard/Configuration.java

```diff
@@ -32,11 +38,27 @@
 
 	final protected Map<Question, Answer> options;
 	final protected String pathOnDisk;
+	protected Answer answr;
+	final protected String taskName;
 
 	@SuppressWarnings("unchecked")
-	public Configuration(Map<?, ?> constraints, String pathOnDisk) {
+	public Configuration(Map<?, ?> constraints, String pathOnDisk, String taskName) throws IOException {

```

Comment:
what would happen if an IOException was thrown? Will the feature not work at all or would it be able to work regardless?

If it still would work, please catch the exception, log it and then forget it.